src/opensearchDatasource.test.ts

```diff
@@ -128,1482 +113,114 @@ describe('OpenSearchDatasource', function (this: any) {
-                                          botness: {
-                                            type: 'float',
-                                          },
-                                          general_algorithm_score: {
-                                            type: 'float',
-                                          },
-                                        },
-                                      },
-                                    },
-                                  },
-                                },
-                              },
-                            },
-                          },
-                        },
-                      },
-                    },
-                  },
-                  overall_vote_score: {
-                    type: 'float',
-                  },
-                  ua_terms_long: {
-                    type: 'keyword',
-                  },
-                  ua_terms_short: {
-                    type: 'keyword',
-                  },
-                },
-              },
-            },
-          },
-        });
-      });
-    });
-
-    it('should return nested fields', async () => {
-      const fieldObjects = await ctx.ds.getFields();
-
-      const fields = _.map(fieldObjects, 'text');
-
-      expect(fields).toEqual([
-        '@timestamp_millis',
-        'classification_terms',
-        'domains',
-        'ip_address',
-        'justification_blob.criterion.keyword',
-        'justification_blob.criterion',
-        'justification_blob.overall_vote_score',
-        'justification_blob.shallow.jsi.sdb.dsel2.bootlegged-gille.botness',
-        'justification_blob.shallow.jsi.sdb.dsel2.bootlegged-gille.general_algorithm_score',
-        'justification_blob.shallow.jsi.sdb.dsel2.uncombed-boris.botness',
-        'justification_blob.shallow.jsi.sdb.dsel2.uncombed-boris.general_algorithm_score',
-        'overall_vote_score',
-        'ua_terms_long',
-        'ua_terms_short',
-      ]);
-    });
-
-    it('should return number fields', async () => {
-      const fieldObjects = await ctx.ds.getFields('number');
-
-      const fields = _.map(fieldObjects, 'text');
-
-      expect(fields).toEqual([
-        'justification_blob.overall_vote_score',
-        'justification_blob.shallow.jsi.sdb.dsel2.bootlegged-gille.botness',
-        'justification_blob.shallow.jsi.sdb.dsel2.bootlegged-gille.general_algorithm_score',
-        'justification_blob.shallow.jsi.sdb.dsel2.uncombed-boris.botness',
-        'justification_blob.shallow.jsi.sdb.dsel2.uncombed-boris.general_algorithm_score',
-        'overall_vote_score',
-      ]);
-    });
-
-    it('should return date fields', async () => {
-      const fieldObjects = await ctx.ds.getFields('date');
-
-      const fields = _.map(fieldObjects, 'text');
-
-      expect(fields).toEqual(['@timestamp_millis']);
-    });
-  });
-
-  describe('When getting fields with backend flow', () => {
```

Comment:
I don't think this test should be removed?

Reply:
Put back these two, got a bit trigger happy